frontend/src/common/components/links/CustomActionLink.scss

```diff
@@ -0,0 +1,8 @@
+.custom-action-link {
+  &#{&} {
+    -webkit-user-select: text;
+    -moz-user-select: text;
+    -ms-user-select: text;
+    user-select: text;
```

Comment:
trying to find more documentation, but I would really hope that Vite's CSS compiler handles cross-browser specific references like this. aka. `-webkit-user-select` vs. `user-select`

Reply:
We can add postcss configuration inside vite config to automatically do this.
Refer: [https://stackoverflow.com/a/71422287](https://stackoverflow.com/a/71422287)

Reply:
ok I'll add this